internal/devserver/main/main.go

```diff
@@ -60,6 +60,18 @@ func main() {
 		preScripts = append(preScripts, livereloadLoaderSnippet)
 	}
 
+	// Include the profiler script if IBAZEL_PROFILER_URL is set.
+	profilerUrl := os.Getenv("IBAZEL_PROFILER_URL")
```

Comment:
nit: I was confused when I read it the first time, I thought this would be the callback URL to send events to.
 `profilerScriptSrc` would be a better name